DCHdigi/dataFormatExtension/driftChamberHit.yaml

```diff
@@ -33,6 +34,24 @@ datatypes:
       - float eDepError               // error measured on eDep [GeV].
       - uint32_t clusterCount         // number of clusters associated to this hit
 
+  extension::DriftChamberDigiV2:
+    Description: "Digitized hit (before tracking) for Drift Chamber v2 (requires data extension)."
+    Author: "A. Tolosa-Delgado, B. Francois, CERN"
+    Members:
+      - uint64_t cellID               // ID of the sensor that created this hit
+      - int32_t type                  // type of the raw data hit
+      - int32_t quality               // quality bit flag of the hit
+      - float time                    // time of the hit [ns]
+      - float eDep                    // energy deposited on the hit [GeV]
+      - float eDepError               // error measured on eDep [GeV]
+      - edm4hep::Vector3d position    // point on the sensitive wire (SW) which is closest to the simhit [mm]
+      - edm4hep::Vector3d position_L  // point on the left to the sensitive wire (SW) [mm]
```

Comment:
This is redundant with the other members, I was actually proposing to add a function in the DD4hep dch extensions to retrieve position_L and position_R from cellID and distanceToWire but I am not even sure it is needed, let's see in the future. Can you remove those two members?

Reply:
I was thinking that kind of functionality is only useful for reconstruction and subsequent steps... maybe an alternative can be placing the functionality in a dedicated header-only file, which will be included by this algorithm, and since it will be installed it can be included by other projects. Actually, we can put there not only that function, but other functions such as the ones calculating the distance to the wire, or the functions that calculate the position of the wire based on the data extension. What do you think?

Reply:
done :)